File: sandbox.client/src/model/Account.ts
export class Account{
  public id : number;
  public client_id : number;
  public money : number;
  public number : string;
  public registered_at: string;


  public static create(a : any): Account {
    const ret = new Account();
    ret.assign(a);
    return ret;
  }

  assign(a : any){
    this.id = a.id;
    this.client_id = a.client_id;
    this.money = a.money;
    this.number =  a.number;
    this.registered_at = a.registered_at;
  }
}

File: sandbox.migration/src/kz/greetgo/learn/migration/core/models/Account.java
package kz.greetgo.learn.migration.core.models;

import com.fasterxml.jackson.annotation.JsonFormat;
import com.fasterxml.jackson.core.JsonFactory;

import java.sql.Date;

public class Account {
    JsonFactory factory = new JsonFactory();
    public Long number;
    public String client_id;
    public String account_number;

    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "dd-MM-yyyy hh:mm")
    public Date registered_at;

}
